ArticleTemplates/assets/js/bootstrap.js

```diff
@@ -34,7 +32,9 @@
             'initMpuPoller',
             'videoPositioning',
             'getArticleHeight',
-            'injectInlineArticleMembershipCreative'
+            'injectInlineArticleMembershipCreative',
+            'getRelatedContentPosition',
+            'setRelatedItemsHeight'
```

Comment:
Minor thing but maybe we can rename `setRelatedItemsHeight` to `setRelatedContentHeight` as it'll help show it's related to the "related content" container we refer to elsewhere in the code.